### DISCUSSION TITLE
Battery capacity limit

### DISCUSSION THREAD

**Discussion by pedromfa** [hif2k1/battery_sim#121] [Ideas]
Would it be possible to add a function to select minimum and maximum battery level? I believe to keep batteries healthy most manufactors suggest to keep the charge between 20% and 80% for maximum life expectancy. This would give us a more accurate study for a future real battery. Thank you. :)

**Comment by Hamish-Blinx** [comment]
There is an issue request for this already (https://github.com/hif2k1/battery_sim/issues/117). It is something we can look at in the future.

**Comment by pedromfa** [comment]
Thank you. That would be brilliant. 😊👍
